src/main/java/com/example/gradecalculator/entities/Subject.java

```diff
@@ -29,6 +29,9 @@ public Subject(String name, String description) {
         this.description = description;
     }
 
+    public Subject(String math) {
```

Comment:
also a constructor that has parameters but does never use those parameters

Reply:
"math" should've been "subject", I fixed it and has usage in test class

Reply:
but values are never assigned. use empty constructor instead